constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,122 @@
+mod MemoryChipletAir
+
+### Helper functions ##############################################################################
+
+fn delta(v: scalar) -> scalar:
+    return v' - v
+
+
+fn get_n0(ctx: scalar, t: scalar) -> scalar:
+    return delta(ctx) * t'
+
+
+fn get_n1(addr: scalar, t: scalar) -> scalar:
+    return delta(addr) * t'
```

Comment:
nit: let's add comments above these functions to explain n0 and n1. You could use the same ones that are inline below